winrt/lib/text/CanvasFontFace.abi.idl

```diff
@@ -76,12 +76,24 @@ namespace Microsoft.Graphics.Canvas.Text
     [version(VERSION)]
     typedef struct CanvasGlyph
     {
-        int Index;
+        UINT16 Index;
         float Advance;
         float AdvanceOffset;
         float AscenderOffset;
     } CanvasGlyph;
 
+    [version(VERSION)]
+    typedef struct CanvasGlyphMetrics
+    {
+      float LeftSideBearing;
```

Comment:
This new result struct contains {LeftSideBearing, RightSideBearing, TopSideBearing, BottomSideBearing}. Could it be reworked to use a glyphbox rect instead? (e.g., a Rect type called Bounds)

We've done a similar thing with how we expose IDWriteTextLayout::GetOverhangMetrics. It's a lot more convenient to work with a 'blackbox' rather than some numbers which are deltas from a blackbox.

There's a comment in the [DirectWrite documentation](https://msdn.microsoft.com/en-us/library/windows/desktop/dd368084(v=vs.85).aspx) with a pointer on how to do this.